src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier supplier = new ColorSupplier();
+
+    private static final int maxNumber = 100;
+
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(maxNumber);
```

Comment:
Random must be a class field
```suggestion
        int number = random.nextInt(maxNumber);
```